src/main/java/com/gluonhq/richtext/RichTextAreaSkin.java

```diff
@@ -240,6 +243,8 @@ private void setup(FaceModel faceModel) {
         getSkinnable().textLengthProperty.bind(viewModel.textLengthProperty());
         getSkinnable().setOnContextMenuRequested(contextMenuEventEventHandler);
         getSkinnable().editableProperty().addListener(this::editableChangeListener);
+        getSkinnable().setOnKeyPressed(this::keyPressedListener);
+        getSkinnable().setOnKeyTyped(this::keyTypedListener);
```

Comment:
Do we really need these even when the control is non-editable?

Reply:
Yes, ctrl+c should work in that case too. I always compare it with a regular TextArea in case of doubt